static/app/views/starfish/components/slideOverPanel.tsx

```diff
@@ -4,30 +4,41 @@ import styled from '@emotion/styled';
 import {motion} from 'framer-motion';
 
 const PANEL_WIDTH = '50vw';
+const PANEL_HEIGHT = '50vh';
 
 type SlideOverPanelProps = {
   children: React.ReactNode;
   collapsed: boolean;
   onOpen?: () => void;
+  slideDirection?: 'Right' | 'Bottom';
 };
 
 export default forwardRef(SlideOverPanel);
 
 function SlideOverPanel(
-  {collapsed, children, onOpen}: SlideOverPanelProps,
+  {collapsed, children, onOpen, slideDirection}: SlideOverPanelProps,
   ref: ForwardedRef<HTMLDivElement>
 ) {
   useEffect(() => {
     if (!collapsed && onOpen) {
       onOpen();
     }
   }, [collapsed, onOpen]);
+  const initial =
+    slideDirection === 'Bottom'
+      ? {opacity: 0, x: 0, y: 0}
+      : {opacity: 0, x: PANEL_WIDTH, y: 0};
+  const final =
+    slideDirection === 'Bottom'
+      ? {opacity: 0, x: 0, y: PANEL_HEIGHT}
+      : {opacity: 0, x: PANEL_WIDTH};
```

Comment:
Since all the numbers here are constants, you could declare `BOTTOM_POSITION_STYLES` and RIGHT_POSITION_STYLES` somewhere above/below rather than instantiating here